cedar-policy-validator/src/json_schema.rs

```diff
@@ -2546,7 +2545,21 @@ mod entity_tags {
                 }
             },
             "actions": {}
-        }});
+        }})
+    }
+
+    #[test]
+    fn roundtrip() {
+        let json = example_json_schema();
+        let json_schema = Fragment::from_json_value(json.clone()).expect("should be valid");
+        let serialized_json_schema = serde_json::to_value(json_schema).expect("should be valid");
+        assert_eq!(json, serialized_json_schema);
+    }
+
+    /// This schema taken directly from the RFC 82 text
+    #[test]
+    fn basic() {
+        let json = example_json_schema();
```

Comment:
This comment probably makes more sense to go on `example_json_schema()`

Reply:
Good catch. Fixed.